tests/test_asyncio/test_search.py

```diff
@@ -1001,3 +1001,16 @@ async def test_search_commands_in_pipeline(modclient: redis.Redis):
     assert "doc2" == res[3][4]
     assert res[3][5] is None
     assert res[3][3] == res[3][6] == ["txt", "foo bar"]
+
+
+@pytest.mark.redismod
+def test_timeout():
```

Comment:
@shacharPash Can you combine the 2 tests into one and rename it to test_query_timeout?